src/github.com/matrix-org/dendrite/federationapi/routing/state.go

```diff
@@ -68,6 +78,26 @@ func GetStateIDs(
 	}
 }
 
+func parseParams(
```

Comment:
I might rename this to `parseEventIDParam` or something more specific if it's just designed to pull out `"event_id"`.

Reply:
Good point.